server/middlewares/stats.js

```diff
@@ -41,44 +41,97 @@ const parseTxBodyOutAmount = (txBody) => {
   }
 }
 
-const incrCountersBy = (key, value) => {
-  const [year, month, day] = getSlicedDate()
-  client.incrby(`${key}:total`, value)
-  client.incrby(`${key}:monthly:${year}-${month}`, value)
-  client.incrby(`${key}:daily:${year}-${month}-${day}`, value)
+const initVisitor = (trackingID) => {
+  return ua({
+    tid: trackingID,
+    cid: 555, //google analytics anonymous visitor
+    strictCidFormat: false, //to accept cid integer instead of UUID v4 format
+  })
+}
+
+const trackPageView = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_TRACKING_ID)
+  const args = {
+    dp: argsObj.path,
+    dt: argsObj.hostname,
+    dh: argsObj.title,
+  }
+
+  visitor.pageview(args).send()
+}
+
+const trackEvent = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_TRACKING_ID)
+  const args = {
+    ec: argsObj.eventCategory,
+    ea: argsObj.eventAction,
+    el: argsObj.eventLabel,
+    ev: argsObj.eventValue,
+  }
+
+  visitor.event(args).send()
 }
 
 const trackVisits = (req, res, next) => {
+  if (!backendConfig.ADALITE_TRACKING_ID) {
```

Comment:
this was originally handled in `index.js`, see `if (backendConfig.REDIS_URL)`. Also, please remove the dead code, i.e. middlewares/basicAuth and statsPage since they are no longer needed. Also, please update .env.example

Reply:
should I add '&& backendConfig.ADALITE_GA_TRACKING_ID' to the if?

Reply:
should I specify our tracking id to example or leave it blank?